canonical-web-and-design/cookie-policy
# Refactor and update

Note: These changes should not be merged until https://github.com/vanilla-framework/vanilla-framework/issues/2121 is resolved.

## Done

- Added Prettier
- Swapped out Jshint for Eslint
- Ditched Gulp in favour for npm scripts
- Upgraded to Vanilla 1.8.0
- Switched out the <p> tag in the cookie notification to a <span> so it will fill the full width of the container (the main bug I came here to address)

## QA

- Pull feature branch
- Run `npm install`
- Run `npm run test` - verify no errors
- Run `npm run build` - verify no errors
- Sanity check README changes


## Screenshot
![screenshot_2019-01-28 canonical cookie policy demo](https://user-images.githubusercontent.com/1413534/51836344-1620e880-22f8-11e9-8a32-111d86e9f251.png)
